### ISSUE TITLE
Quest: A Dark Threat Looms (7) bugged #3274

### ISSUE THREAD

**Issue by Navashed** [elysium-project/bug-tracker#100]
From the old bug tracker:

> When accepting the last quest in the quest chain, given from the explosive keg, the quest disappears and cannot be taken from the keg. 2017-11-24

**Comment by Navashed** [comment]
Reproduced.

This will occur if you are level 16 or 17 when you hand in the quest.

Quest 280 - A Dark Threat Looms (6) requires level 16.
Quest 283 - A Dark Threat Looms (7) requires level 18.
Object 1585 - Explosive Charge

I was able to reproduce the level 18 quest being "offered" to a level 17 character:
![48348c95a8d7ad36935d8347e7271048](https://user-images.githubusercontent.com/2709545/33346006-31d52532-d45c-11e7-9121-a211aabc32db.jpg)
When Accept is pressed, the player is told they do not meet the level requirement. Object 1585 - Explosive Charge did not offer the quest again, even when I returned at level 18+.

This is an issue. Any player who hands in Quest 280 while under the level requirement for Quest 283 can never pick up Quest 283 because Object 1585 will not interact with them.
